installer/sharch_body.sh

```diff
@@ -25,13 +25,20 @@ fi
 
 echo " OK."
 
+image_size=$(du "$0" | awk '{print $1}')
```

Comment:
Can you check du returns aligned size like 8,16,32,... this would ease in getting mount size

Reply:
@antony-rheneus du  return without  aligned .